packages/shared-ux/button_toolbar/src/index.ts

```diff
@@ -16,3 +16,7 @@ export type {
 
 export { ToolbarPopover } from './popover';
 export type { ToolbarPopoverProps } from './popover';
+
+export { Toolbar } from './toolbar';
+export type { ToolbarProps } from './toolbar';
+export type { Button } from './toolbar';
```

Comment:
I think this would better be called `ToolbarButton` or something.

Reply:
I can get behind that thanks! [3247796](https://github.com/elastic/kibana/pull/137224/commits/324779666ce1e58346ae84b3ef6ac4d08d7ecdbc)